auction-server/src/opportunity/service/get_quote.rs

```diff
@@ -349,12 +383,12 @@ impl Service<ChainTypeSvm> {
 
         // Find winner bid:
         match input.quote_create.tokens {
-            entities::QuoteTokens::InputTokenSpecified { .. } => {
-                // highest bid = best (most output token returned)
+            entities::QuoteTokens::UserTokenSpecified { .. } => {
+                // highest bid = best (most searcher token returned)
                 bids.sort_by(|a, b| b.amount.cmp(&a.amount));
             }
-            entities::QuoteTokens::OutputTokenSpecified { .. } => {
-                // lowest bid = best (least input token consumed)
+            entities::QuoteTokens::SearcherTokenSpecified { .. } => {
+                // lowest bid = best (least user token consumed)
```

Comment:
this was wrong previously?

Reply:
depends on perspective and what we would have ended up doing with the `QuoteCreate` field meanings. it was definitely confusing, bc of the relative naming. this is a lot clearer.